arrayfire/array.py

```diff
@@ -20,6 +21,63 @@
 from .index import *
 from .index import _Index4
 
+_is_running_in_py_charm = "PYCHARM_HOSTED" in os.environ
+
+_display_dims_limit = None
+
+def set_display_dims_limit(*dims):
+    """
+    Sets the dimension limit after which array's data won't get
+    presented to the result of str(arr).
+
+    Default is None, which means there is no limit.
+
+    Parameters
+    ----------
+    *dims : dimension limit args
+
+    Example
+    -------
+    set_display_dims_limit(10, 10, 10, 10)
+
+    """
+    global _display_dims_limit
+    _display_dims_limit = dims
+
+def get_display_dims_limit():
+    """
+    Gets the dimension limit after which array's data won't get
+    presented to the result of str(arr).
+
+    Default is None, which means there is no limit.
+
+    Returns
+    -----------
+        - tuple of the current limit
+        - None is there is no limit
+
+    Example
+    -------
+    get_display_dims_limit()
+    # None
+    set_display_dims_limit(10, 10, 10, 10)
+    get_display_dims_limit()
+    # (10, 10, 10, 10)
+
+    """
+    return _display_dims_limit
+
+def _in_display_dims_limit(dims):
+    if _is_running_in_py_charm:
+        return False
+    print(_display_dims_limit)
```

Comment:
Yeah, I didn't have time to finish the commit properly. Lemme fix things up tomorrow.